### DISCUSSION TITLE
Inverse Kinematics

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#134] [Help / Questions]
Does anyone know of any javascript libraries that perform inverse kinematics that would integrate well with `enable3d`?

**Comment by vitalspace** [comment]
check https://threejs.org/examples/webgl_camera_cinematic.html

**Reply by N8python** [reply]
Not camera cinematics - inverse kinematics.

**Reply by vitalspace** [reply]
maybe https://jsantell.github.io/THREE.IK/ 🤔

**Reply by N8python** [reply]
Maybe
